TLM/TLM/Custom/AI/CustomTransportLineAI.cs

```diff
@@ -4,70 +4,111 @@
     using API.Traffic.Enums;
     using ColossalFramework;
     using CSUtil.Commons;
-    using Custom.PathFinding;
+    using JetBrains.Annotations;
+    using PathFinding;
     using RedirectionFramework.Attributes;
-    using State;
+    using State.ConfigData;
     using UnityEngine;
 
+    // TODO inherit from NetAI (in order to keep the correct references to `base`)
     [TargetType(typeof(TransportLineAI))]
-    public class CustomTransportLineAI : TransportLineAI { // TODO inherit from NetAI (in order to keep the correct references to `base`)
```

Comment:
comment is missing but still valid

Reply:
It is moved 2 lines up
![image](https://user-images.githubusercontent.com/288724/61579535-86216600-ab06-11e9-9e07-46cc8932240f.png)